src/indexing.jl

```diff
@@ -48,6 +47,31 @@ end
     return v
 end
 
+# Slice
+@inline function Base.getindex{dim}(S::Tensor{2,dim}, i::Colon, j::Int)
+    @boundscheck checkbounds(S,i,j)
+    @inbounds v = Vec{dim}(S.data[(1+dim*(j-1)):dim*j])
+    return v
+end
+
+@inline function Base.getindex{dim}(S::Tensor{2,dim}, i::Int, j::Colon)
+    @boundscheck checkbounds(S,i,j)
+    @inbounds v = Vec{dim}(S.data[i:dim:(dim*(dim-1) + i)])
+    return v
+end
+
+@inline function Base.getindex{dim}(S::SymmetricTensor{2,dim}, i::Colon, j::Int)
+    @boundscheck checkbounds(S,i,j)
+    idx = Array(Int,dim)
```

Comment:
It will be like this then:
```jl
julia> A = rand(SymmetricTensor{2,2})
2×2 ContMechTensors.SymmetricTensor{2,2,Float64,3}:
 0.289876  0.785343
 0.785343  0.971371

julia> A[:,1]
2-element ContMechTensors.Tensor{1,2,Float64,2}:
 0.289876
 0.785343

julia> A[:,3]
ERROR: BoundsError: attempt to access (0.28987552553562224,0.78534345301397,0.9713714024100475)
  at index [4]
 in macro expansion at ./REPL[2]:9 [inlined]
 in getindex(::ContMechTensors.SymmetricTensor{2,2,Float64,3}, ::Colon, ::Int64) at ./REPL[2]:2
```